sql/updates/README.md

```diff
@@ -28,7 +28,7 @@ to consider.
    commands that change or remove objects. In some cases,
    modifications of metadata are also necessary.
 2. `DROP FUNCTION` needs to be idempotent. In most cases that means
-   commands should have an `IF NOT EXISTS` clause. The reason is that
+   commands should have an `IF EXISTS` clause. The reason is that
    some modfiles might try to, e.g., `DROP` functions that aren't
    present because they only exist in an intermediate version of the
    database, which is skipped over.
```

Comment:
Can you add a 5th bullet to this list with the same note that you have in `aggregates.sql`? Basically something like "If a new aggregate was added in the new version, it should also be created in the modfile leading to that new version"?

Reply:
updated and rebased